R/spec.R

```diff
@@ -66,7 +66,11 @@ spec_validate_arg <- function(call, signaller) {
   }
 
   if (length(arg) != 1L) {
-    abort("Internal error: `what` can't refer to more than one argument.")
+    fn <- as_string(node_car(call))
```

Comment:
I'd use `as_label()` instead of `as_string()` to get a more useful error message in case the CAR isn't a symbol. (Not sure if it's actually possible here, just to be defensive)